### DISCUSSION TITLE
Does this still work?

### DISCUSSION THREAD

**Discussion by jpad-freedomfi** [tonesto7/fordpass-scriptable#134] [Q&A]
Hi All, 

Does this still work? I would love to get it working but failing on iOS16.3. 

I get scriptable installed
I get the ford widget install tool pasted and run
but when I go to login to Ford Pass account it does not find my vehicle. 
My ford pass account works for FDRS and ford pass app and website. So I think I have correct login info and working ford pass account. 

Any ideas what I'm doing wrong?

**Comment by hweissig** [comment]
> Hi All, 
> 
> 
> 
> Does this still work? I would love to get it working but failing on iOS16.3. 
> 
> 
> 
> I get scriptable installed
> 
> I get the ford widget install tool pasted and run
> 
> but when I go to login to Ford Pass account it does not find my vehicle. 

Same here. After adding my VIN manually, I am getting an error from the script:

![image](https://user-images.githubusercontent.com/115736834/218939752-30ac77e4-a6c5-4cf4-afa2-aab4a0929ce0.png)

**Comment by helpmerick** [comment]
I'm in same boat. Installed perfectly, it seemed. Needed to manually add VIN, and then got same error. iPhone 12 Mini , iOS 16.4.1

**Comment by ccanalia82** [comment]
@tonesto7 Same boat here, iPhone 12Pro.  I also had to manually add my VIN and everything worked until I tried to run the widget.

**Reply by MaDDoGGeRR22** [reply]
Had the same issue. I heard that Ford Pass updated their software to block 3rd party apps from accessing any info (along with controlling vehicles with said info)

don’t know if that is true, but @tonesto7 will hopefully reply and help us figure this issue out

**Comment by sobmartin45** [comment]
Doesn’t work on IPadOS 17 beta 5. Creates the widget but will not open to screen

**Comment by stickyfingers2k** [comment]
I would also like to know if this is supported.  I assume as others have said that Ford may have changed the API to block 3rd party apps connecting to their systems. That's a shame.
